### PR TITLE
Enable family_connect_registration_pod

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by justinvdm** [comment]
:+1:, lets land when travis is happy (doing a release now, will restart travis build afterwards)

**Comment by KaitCrawford** [comment]
:+1: on the changes. have you rebuilt travis?

File: casepro/settings_common.py [hunk 1]
```diff
@@ -496,31 +496,3 @@
...
 
 CELERY_TIMEZONE = 'UTC'
-
-# Pods
-
-INSTALLED_APPS += (
-    'casepropods.family_connect_registration.plugin.RegistrationPlugin',
-)
-
-PODS = [{
```
Comment (KaitCrawford) [inline casepro/settings_common.py]: I think you need to leave the empty list here for travis to pass

Comment (KaitCrawford) [inline casepro/settings_common.py]: I'll do it quickly though

**Pull request merged**